### PR TITLE
Requesting Changes in `REXSushiFarmMarket.sol`

### PR DESCRIPTION
**Pull request by rashtrakoff**
There are some inconsistencies in the `_swapAndDeposit` function. These are:

 - Function is marked as public even though internal function naming convention has been used. Need to check for security issues.
 - Function parameters aren't used in the function at all.
 
I request @mikeghen  to make necessary changes. Note that `REXMarket.sol` has also been changed according to the style guide.

### CODE REVIEW

File: test/helpers.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    for (let i = 0; i < accounts.length; ++i) {
+        await network.provider.request({
+            method: "hardhat_impersonateAccount",
+            params: [accounts[i]]
+        });
+
+        await network.provider.send("hardhat_setBalance", [
+            accounts[i],
+            ethers.utils.hexValue(1000),
+            // hexValue(parseEther("1000")),
```
Comment (shreyaspapi) [inline test/helpers.ts:26]:
```suggestion
```

**Review by shreyaspapi — COMMENTED** [review]
Do we need all the commented out code, also the console logs?

File: test/helpers.ts (hunk 1 above)
Comment (rashtrakoff) [reply test/helpers.ts:26]: This isn't really the scope of this PR. This PR is strictly related to `REXSushiFarmMarket.sol` and it's dependencies.

**Comment by rashtrakoff** [comment]
Changes have been communicated and being made in my fork.

**Pull request closed without merging**